spring-cloud-vault-core/src/main/java/org/springframework/cloud/vault/VaultClient.java

```diff
@@ -15,45 +15,67 @@
  */
 package org.springframework.cloud.vault;
 
-import java.net.URI;
-import java.util.Map;
+import lombok.Getter;
+import lombok.Setter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 import org.springframework.http.HttpEntity;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpMethod;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.ClientHttpRequestFactory;
 import org.springframework.util.Assert;
 import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.HttpStatusCodeException;
 import org.springframework.web.client.RestTemplate;
 
-import lombok.Getter;
-import lombok.Setter;
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
 
 /**
  * Vault client. This client reads data from Vault.
  *
  * @author Spencer Gibb
  * @author Mark Paluch
+ * @author Stuart Ingram
  */
 public class VaultClient {
 
 	public static final String API_VERSION = "v1";
 	public static final String VAULT_TOKEN = "X-Vault-Token";
 
+	public final static String INITIALIZE_URL_TEMPLATE = "sys/init";
+	public final static String UNSEAL_URL_TEMPLATE = "sys/unseal";
+	public final static String SEAL_STATUS_URL_TEMPLATE = "sys/seal-status";
+	public final static String HEALTH_URL_TEMPLATE = "sys/health";
+
 	@Setter
 	@Getter
 	private RestTemplate restTemplate;
 
-	public VaultClient() {
-		this(new RestTemplate());
+	@Getter
```

Comment:
Agreed.  I'll look to the VaultTemplate to facilitate this functionality moving forwards.